tensorbay/utility/common.py

```diff
@@ -300,3 +300,50 @@ def wrapper(*arg: Any, **kwargs: Any) -> Any:
             return func(*arg, **kwargs)
 
         return wrapper  # type: ignore[return-value]
+
+
+class Disable:  # pylint: disable=too-few-public-methods
+    """A decorator for the function which is disable temporarily.
+
+    Arguments:
+        since: The version the function is disable temporarily.
+        back: The version the function will be able again.
+        reason: The reason that the function is disable temporarily.
+
+    """
+
+    def __init__(
```

Comment:
for api consistency, make the argument of `__init__` like:
```python
def __init__(self, *, since: str, enabled_in: Optional[str], reason: Optional[str]):
```